### PR TITLE
Ed 1289 initial

### PR DESCRIPTION
**Pull request by richtier**

### CODE REVIEW

File: directory_header_footer/context_processors.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        'sso_is_logged_in': request.sso_user is not None,
+        'sso_login_url': '{0}?next={1}'.format(settings.SSO_LOGIN_URL, url),
+        'sso_register_url': settings.SSO_SIGNUP_URL,
+        'sso_logout_url': settings.SSO_LOGOUT_URL,
+        'sso_profile_url': settings.SSO_PROFILE_URL,
+    }
+
+
+def header_footer_context_processor(request):
+    active_classes = getattr(settings, 'HEADER_FOOTER_CSS_ACTIVE_CLASSES', {})
```
Comment (konradko) [inline directory_header_footer/context_processors.py]: what happens when it returns `{}`?

File: directory_header_footer/context_processors.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+from django.conf import settings
+
+
+def sso_processor(request):
+    url = request.build_absolute_uri()
+    sso_password_reset_url = getattr(settings, 'SSO_PASSWORD_RESET_URL', None)
```
Comment (konradko) [inline directory_header_footer/context_processors.py]: why is that one optional and the rest not?

File: requirements.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
+Django==1.9.10
```
Comment (konradko) [inline requirements.txt]: that's old unsecure version, we're using 1.9.13 I think

File: directory_header_footer/context_processors.py (hunk 2 above)
Comment (richtier) [inline directory_header_footer/context_processors.py]: `SSO_PASSWORD_RESET_URL` is only used by sso profile.

I will move `SSO_PASSWORD_RESET_URL` to sso profile context processor.

File: directory_header_footer/context_processors.py (hunk 1 above)
Comment (richtier) [inline directory_header_footer/context_processors.py]:
```
<a class="{% if HEADER_FOOTER_CSS_ACTIVE_CLASSES.fab %}navigation-main-link--active{% endif %}">Find a buyer</a>```

if `HEADER_FOOTER_CSS_ACTIVE_CLASSES` is {} then the template code will evaluate to false and not use the active class.

**Pull request merged**